pkg/render/intrusiondetection/dpi/dpi.go

```diff
@@ -321,6 +344,28 @@ func (d *dpiComponent) dpiClusterRoleBinding() *rbacv1.ClusterRoleBinding {
 	}
 }
 
+func (d *dpiComponent) dpiLinseedAccessClusterRoleBinding() *rbacv1.ClusterRoleBinding {
+	return &rbacv1.ClusterRoleBinding{
+		TypeMeta: metav1.TypeMeta{Kind: "ClusterRoleBinding", APIVersion: "rbac.authorization.k8s.io/v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   fmt.Sprintf("%s-extra", DeepPacketInspectionNamespace),
```

Comment:
I don't thinks this name is right - it doesn't explain what this CRB does. For example, `tigera-dpi-linseed-permissions`

I'd also like to see a comment on this function explaining when this CRB is expected to exist and why. I think the answer to that is "This binding should exist in management and standalone clusters - essentially any cluster running Linseed.  We need this for authz checks performed by Linseed when receiving requests from DPI"